crmprtd/__init__.py

```diff
@@ -167,7 +169,12 @@ def run_data_pipeline(download_func, normalize_func, download_args):
                 f.write(chunk)
 
     rows = [row for row in normalize_func(download_iter)]
-    for row in rows:
-        print(row)
-    # observations = [align(row) for row in rows]
+
+    engine = create_engine(args.connection_string)
+    Session = sessionmaker(engine)
+    sesh = Session()
+
+    observations = [align(sesh, row) for row in rows]
+    for ob in observations:
```

Comment:
Want to add that filter for `None` values here that we discussed?

Reply:
Oops, forgot to add that in. My mistake.